### PR TITLE
Basic flair implementation

### PR DESCRIPTION
**Pull request by carteryu**
Need to work on UI, but this is the basic implementation for setting flairs. Will dive deep when designs are out.

### CODE REVIEW

File: civilapp/settings/SettingsScreen.js [hunk 1]
```diff
@@ -1,37 +1,156 @@
...
+      key: 'Cincinnati_Bengals',
+      link: require('../sports/assets/NFL/Cincinnati_Bengals_logo.png'),
+    },
+    {
+      key: 'Cleveland_Browns',
+      link: require('../sports/assets/NFL/Cleveland_Browns_logo.png'),
+    },
+    {
+      key: 'Cincinnati_Bengals',
+      link: require('../sports/assets/NFL/Cincinnati_Bengals_logo.png'),
```
Comment (EphemeralX) [inline civilapp/settings/SettingsScreen.js]: Duplicate from line 57

File: civilapp/settings/SettingsScreen.js [hunk 2]
```diff
@@ -1,37 +1,156 @@
...
 class SettingsScreen extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      userId: '',
+      showSuccess: false,
+    }
+  }
+  //TODO: finish populating the data array (prob NBA/Hockey/Soccer leagues too before launch? IDK)
+  data = [
```
Comment (EphemeralX) [inline civilapp/settings/SettingsScreen.js]: nit: We should put this in a config file imo

Comment (EphemeralX) [reply civilapp/settings/SettingsScreen.js]: also it looks like you're missing some entries based on the asset files in this commit.

File: civilapp/settings/SettingsScreen.js (hunk 1 above)
Comment (carteryu) [reply civilapp/settings/SettingsScreen.js]: Oops

**Pull request merged**